lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -54,11 +87,16 @@ def extra_attributes
         end
 
         def pii_from_doc
-          return {} unless true_id_product&.dig(:AUTHENTICATION_RESULT).present?
+          return {} unless true_id_product&.dig(:IDAUTH_FIELD_DATA).present?
 
-          true_id_product[:AUTHENTICATION_RESULT].select do |k, _v|
-            PII_DETAILS.include? k
+          pii = {}
+          PII_INCLUDES.each do |key, value|
+            pii[value] = true_id_product[:IDAUTH_FIELD_DATA][key]
```

Comment:
Maybe call `value` something like `mapped_key` or something to make it clearer, it's not a value?

```suggestion
          PII_INCLUDES.each do |key, mapped_key|
            pii[mapped_key] = true_id_product[:IDAUTH_FIELD_DATA][key]
```

Reply:
Yeah, I changed it to be more expressive on both parameters.